plugins/woocommerce/includes/admin/class-wc-admin-marketplace-promotions.php

```diff
@@ -0,0 +1,266 @@
+<?php
+/**
+ * Addons Page
+ *
+ * @package  WooCommerce\Admin
+ * @version  2.5.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Admin_Marketplace_Promotions class.
+ */
+class WC_Admin_Marketplace_Promotions {
+
+	const TRANSIENT_NAME = 'wc_addons_marketplace_promotions';
+	public static string $locale = 'fr_FR';
```

Comment:
🇫🇷 ? I like it but maybe it shouldn't be there 😉

Reply:
Thanks, this was debugging code! Removed in c8fc24668b79740d10263a092edef91da68c3dbf.